tests/unit/buttons.js

```diff
@@ -27,5 +27,17 @@ define('tests/unit/buttons',
                     assert.equal(buttons.getBtnText(app), 'pay me $10.25');
                 });
             }));
+
+        it('is "Install" when paid and purchased', h.injector(
+                h.mockSettings({
+                    meowEnabled: true,
+                    model_prototypes: {app: 'slug'},
+                })
+            ).run(['buttons', 'core/user'], function(buttons, user) {
+                var app = {id: 123, price: '10.25', price_locale: '$10.25'};
+                assert.equal(buttons.getBtnText(app), 'Install for $10.25');
+                user.update_purchased(app.id);
+                assert.equal(buttons.getBtnText(app), 'Install');
```

Comment:
I'd prefer for the button text to be stored in a `require`-able constant that's used in both `getBtnText` and this test, so the test doesn't automatically fail if we change the copy. I don't like magic strings.